### PR TITLE
Privitizing

### PR DESCRIPTION
**Pull request by bluong**
There are several instance variables that are protected that should be instead private.  Few cleanups that we would want to do before 1.0

QE: Regression pass please.  The JS got affected functionally in very small ways, specifically with how `Axis.Time` sets up its DOM elements and how `Plots.ClusteredBar` lays out the bars for the cluster, but these changes should hopefully not affect you.

### CODE REVIEW

File: src/components/plots/stackedAreaPlot.ts [hunk 1]
```diff
@@ -97,10 +97,6 @@ export module Plots {
       return this;
     }
 
-    protected _wholeDatumAttributes() {
```
Comment (jtlan) [inline src/components/plots/stackedAreaPlot.ts]: Was this just unused? How did that happen?

**Comment by jtlan** [comment]
[+1] Probably a regression pass is sufficient, if that's even necessary.

File: src/components/plots/stackedAreaPlot.ts (hunk 1 above)
Comment (bluong) [inline src/components/plots/stackedAreaPlot.ts]: I likely forgot this when i was cleaning up areaPlot

**Comment by bluong** [comment]
Let's be safe unless bandwidth gets clogged

**Comment by crmorford** [comment]
Axis.Time looks good in the regression pass, wanna check out clustered bar a little bit.

**Pull request merged**